.ado/release.yml

```diff
@@ -122,17 +126,27 @@ extends:
                   - task: UsePythonVersion@0
                     inputs:
                       versionSpec: $(PYTHON_VERSION)
+                    displayName: Install Python $(PYTHON_VERSION)
                     condition: and(succeeded(), ne(variables['arch'], 'aarch64'))
 
+                  # UsePythonVersion@0 does not support aarch64, so we install Python manually
+                  # using tdnf on Azure Linux 3.0 aarch64
+                  # We also need alias python3 as python
+                  - script: |
+                      sudo tdnf install python3 python3-pip -y
```

Comment:
Should this use a more specific version that `python3` to ensure it matches the `PYTHON_VERSION` that is used by other platforms? (Else I assume this might pick up 3.13 versus the 3.12 specified)